prompts.js

```diff
@@ -13,36 +10,4 @@ module.exports = [
       },
     ],
   },
-  {
-    when: (answers) => answers.type === 'init',
-    name: 'semver',
-    type: 'input',
-    default: '>=5.3.0',
-    message: `What storybook version do you want? ${chalk.yellow('(Please specify semver range)')}`,
-    validate: (input) => {
-      if (input === '' || !semver.validRange(input)) {
-        return 'Given semver range is not valid.';
-      }
-
-      if (!semver.prerelease(input) && !semver.intersects(input, '>=4.1.0')) {
-        return 'Minimum supported storybook version is 4.1.0';
-      }
-
-      return true;
-    },
-  },
```

Comment:
I think we should keep this storybook version question and change the minimum storybook version to 6

Reply:
Done